Added details of how to get a Forms license that will work in FIPS-compliant environments. (#5984)
umbraco/UmbracoDocs: 13/umbraco-forms/installation/the-licensing-model.md

{% hint style="warning" %}
This will also change the location for other Umbraco-related licenses in this project.
{% endhint %}

> ## Note on Federal Information Processing Standards (FIPS) Compliant Environments

Reviewer: ```suggestion
## Federal Information Processing Standards (FIPS) Compliant Environments
```

The algorithm used to decrypt Forms licenses is not supported on locked down FIPS compliant environments, such as those used in the defense industry.

> If you are in this situation and unable to resolve via configuration of the environment, please reach out to Umbraco support.

Reviewer: ```suggestion
If you are in this situation and unable to resolve it via configuration of the environment, please reach out to Umbraco support.
```

We have the possibility of generating and providing Forms licenses using alternate algorithms.

> These can be used if you reference `Umbraco.Licensing` version `13.0.1` or higher.

Reviewer: ```suggestion
{% hint style="info" %}
You can use this configuration if you're referring to `Umbraco.Licensing` version `13.0.1` or higher.
{% endhint %}
```

> And apply the following configuration with the appropriate algorithm - `DES` (the default), `TripleDES` or `AES`:

Reviewer: ```suggestion
Apply the following configuration with the appropriate algorithm - `DES` (the default), `TripleDES`, or `AES`:
```
